zellij-utils/src/input/options.rs

```diff
@@ -24,6 +24,9 @@ pub struct Options {
     /// subdirectory of config dir
     #[structopt(long, parse(from_os_str))]
     pub layout_dir: Option<PathBuf>,
+    #[structopt(long)]
+    #[serde(default)]
+    pub disable_mouse_mode: bool,
```

Comment:
I couldn't get this to work. I did `zellij options --disable-mouse-mode` and the mouse was still functional. Am I doing something wrong or misunderstanding?

Reply:
It should work in the config file already with `disable_mouse_mode: true`, but the merged config is not used yet.